chat-commands.js

```diff
@@ -3409,6 +3409,21 @@ exports.commands = {
 				group: targetUser.group,
 				rooms: roomList,
 			};
+			if (targetUser in room.users && user.can('addhtml', null, room)) {
```

Comment:
I was just pointing that out. But I still don't want bots knowing who I battle with.
Why should bots have privileges again?